packages/vuepress-theme-odyssey/global-components/Description.vue

```diff
@@ -21,7 +22,12 @@ export default {
     tag: {
       default: "div",
       type: String
-    }
+    },
+    layout: {
+      type: String,
+      required: false,
+      default: null
```

Comment:
would an empty string make more sense for a default if we're enforcing a type of `String` above on line 27? I

Reply:
@arnoldsandoval-okta can you speak to this? I'm replicating your work from other components here to add the same layout option.

Reply:
Yes, it would.